packages/galileo-cli/src/lib/prompts/index.ts

```diff
@@ -297,6 +331,125 @@ namespace galileoPrompts {
     };
   }
 
+  export async function ragEmbeddingModel(
```

Comment:
this fn doing enough to possibly need an overview comment

Reply:
No used outside of the cli internals (not public) so will leave for now - but in future after we stabalize the cli will be good to add docs for all